imgix/validators.py

```diff
@@ -0,0 +1,100 @@
+from .constants import IMAGE_MAX_WIDTH as MAX_WIDTH
+from .constants import IMAGE_ZERO_WIDTH as ZERO_WIDTH
+
+
+def validate_min_width(value):
+    """
+    Validate the minimum width value.
+
+    This function ensures that the `value`:
+    * is of type `int`, or
+    * is of type `float`, and
+    * is greater than the `ZERO_WIDTH`
+
+    This function is used to ensure custom minimum widths are within
+    a valid range. Here, valid means that the minimum width value must
+    an int, or a float, and be greater than zero.
+
+    Raises
+    ------
+    AssertionError
+        This function is designed to fail upon invalid input to
+        prevent the propagation of invalid state.
+
+    Parameters
+    ----------
+    value : float, int
+        A valid `value` must be a positive numerical value.
+    """
+    invalid_width_error = 'error: `min_width` must be a positive ' \
+        '`float` or `int`'
+    assert isinstance(value, (float, int)), invalid_width_error
+
+    invalid_min_error = 'error: `min_width` must be greater than zero'
+    assert value > ZERO_WIDTH, invalid_min_error
+
+
+def validate_max_width(value):
```

Comment:
```suggestion
def validate_max_width(max_width):
```

Reply:
While I do like how explicit `min_width` is, in this particular case I feel that it's redundant (i.e. not DRY). We shouldn't repeat ourselves if we don't have to. Moreover, using this pattern could help in the future by not having multiple similarly named values within function bodies.